doc/source/data/examples/ocr_example.ipynb

```diff
@@ -0,0 +1,545 @@
+     "data": {
+      "text/plain": [
+       "RayContext(dashboard_url='127.0.0.1:8265', python_version='3.7.4', ray_version='1.13.0', ray_commit='e4ce38d001dbbe09cd21c497fedd03d692b2be3e', address_info={'node_ip_address': '127.0.0.1', 'raylet_ip_address': '127.0.0.1', 'redis_address': None, 'object_store_address': '/tmp/ray/session_2022-07-04_14-35-16_950060_89285/sockets/plasma_store', 'raylet_socket_name': '/tmp/ray/session_2022-07-04_14-35-16_950060_89285/sockets/raylet', 'webui_url': '127.0.0.1:8265', 'session_dir': '/tmp/ray/session_2022-07-04_14-35-16_950060_89285', 'metrics_export_port': 60416, 'gcs_address': '127.0.0.1:61663', 'address': '127.0.0.1:61663', 'node_id': 'b6c981243d51558d13e4290f0f63552a6126f8a8d9e472baafe9dd5b'})"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Import ray and initialize a local Ray cluster.\n",
+    "import ray\n",
+    "ray.init()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "785f4992",
+   "metadata": {},
+   "source": [
+    "### Running the OCR software on the data\n",
+    "\n",
+    "We can now use the {meth}`ray.data.read_binary_files <ray.data.read_binary_files>` function to read all the images from S3. We set the `include_paths=True` option to create a dataset of the S3 paths and image contents. We then run the {meth}`ds.map <ray.data.Dataset.map>` function on this dataset to execute the actual OCR process on each file and convert the screen shots into text. This will create a tabular dataset with columns `path` and `text`, see also [](transform_datasets_row_output_types).\n",
+    "\n",
+    "````{note}\n",
+    "If you want to load the data from a private bucket, you have to run\n",
+    "```python\n",
+    "import pyarrow.fs\n",
+    "\n",
+    "ds = ray.data.read_binary_files(\"s3://<bucket>/<folder>\",\n",
+    "    include_paths=True,\n",
+    "    filesystem=pyarrow.fs.S3FileSystem(\n",
+    "        access_key=\"...\",\n",
+    "        secret_key=\"...\",\n",
+    "        session_token=\"...\"))\n",
+    "```\n",
+    "````"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "81b517d3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-07-04 14:35:53,683\tWARNING read_api.py:256 -- The number of blocks in this dataset (3) limits its parallelism to 3 concurrent tasks. This is much less than the number of available CPU slots in the cluster. Use `.repartition(n)` to increase the number of dataset blocks.\n",
+      "Read->Map: 100%|██████████| 3/3 [00:07<00:00,  2.34s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from io import BytesIO\n",
+    "from PIL import Image\n",
+    "import pytesseract\n",
+    "\n",
+    "def perform_ocr(data):\n",
+    "    path, img = data\n",
+    "    return {\"path\": path, \"text\": pytesseract.image_to_string(Image.open(BytesIO(img)))}\n",
```

Comment:
To keep the wrapped line from making the indentation confusing, could we format this returned dictionary so it doesn't automatically wrap?